.circleci/config.yml

```diff
@@ -909,16 +914,17 @@ jobs:
     - run:
         name: Check whether to run job
         command: |
-          if [[ -f "${WORKSPACE_ROOT}/pr-metadata/labels/skip-integration-tests" ]]; then
-            echo "Skipping job with skip-integration-tests label."
-            "${CI_ROOT}/safe-halt.sh"
-            exit
-          fi
+          dockerized=<< parameters.dockerized >>
```

Comment:
As far as I can tell `dockerized` is not being used.
```suggestion
```

Reply:
Done